### PR TITLE
CLOUDP-225724 Use Github Actions Job Summary instead of Gist

### PR DESCRIPTION
**Pull request by fmenezes**
* Update setup step to use `setup/au_setup.ps1`
* Use Github Actions Job Summary instead of Gist

### CODE REVIEW

File: .github/workflows/run.yml [hunk 1]
```diff
@@ -14,11 +14,32 @@ jobs:
...
+          $profileInstall | Out-File $profile
+          ./setup/au_setup.ps1
       - run: ./update_all.ps1
+        shell: powershell
         env:
           au_Push: true
-          gist_id: 401de57398b774a70ad4dc5d46aab760
           github_user_repo: ${{ env.GITHUB_REPOSITORY }}
-          github_api_key: ${{ secrets.GH_TOKEN_WITH_GIST }}
+          github_api_key: ${{ secrets.GITHUB_TOKEN }}
```
Comment (fmenezes) [inline .github/workflows/run.yml:40]: now we only need github actions for this

**Review by gssbzn — APPROVED** [review]

File: .github/workflows/run.yml [hunk 2]
```diff
@@ -14,11 +15,27 @@ jobs:
     steps:
       - name: Checkout repository
         uses: actions/checkout@v4
-      - run: choco install -y au
+      - name: setup
```
Comment (andreaangiolillo) [inline .github/workflows/run.yml:18]: Could we update the name of the action to be more meaningful? I believe this is for windows but more context would be great.

**Review by andreaangiolillo — APPROVED** [review]
LGTM. I wasn't aware of this fork 😅 . Where can I see more info on why this fork is needed and how is used for? Thanks!

**Comment by fmenezes** [comment]
> LGTM. I wasn't aware of this fork 😅 . Where can I see more info on why this fork is needed and how is used for? Thanks!

this is new, basically, we are moving chocolatey out of mongodb/mongodb-atlas-cli into here. After the next version is out and this gets kicked in I can give more context.

**Pull request merged**